src/shared/os-util.c

```diff
@@ -117,3 +117,26 @@ int load_os_release_pairs(const char *root, char ***ret) {
 
         return load_env_file_pairs(f, p, ret);
 }
+
+int load_os_release_pairs_container_prefix_env(const char *root, char ***ret) {
+        _cleanup_strv_free_ char **os_release_pairs = NULL;
+        char **p, **q, *line;
```

Comment:
please declare variables at the innermost scope possible, i.e. line should be declared inside the loop